### PR TITLE
Migrate pem into wasting doc

### PR DESCRIPTION
**Pull request by beatrixkh**

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]
GitHub is making it really hard to review this PR.  I'll try to look at it carefully after it is merged, too.

**Comment by Ninicorn** [comment]
Somehow, I can't review directly. Below are some suggestions: 

1) we should note the source state for the **incidence** we derive from the markov model (just like you have it for the remission) in table: 'Wasting model input data sources'. For eg. incidence of mild wasting from tmrel; incidence of MAM from mild; incidence of SAM from MAM. This is so we do not confuse it with other definitions of incidence of SAM or MAM (eg. from a 'non-SAM' or a 'non-MAM' pool respectively). This just helps us keep track of what the denominator is. 

2) In the PEM Data Sources and Definitions, the sid and definitions are reversed (I don't know if it matters so much since you've defined it in the hierarchy). It should be s199 - severe without oedema and s2036 = severe with oedema. 

2) minor detail: there is an extra f2 in the diagram under vivarium modelling strategy

Thank you for your hard work!

File: docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst [hunk 1]
```diff
@@ -0,0 +1,997 @@
...
+     - Description
+     - Notes
+   * - MAM sequelae
+     - 
+     - {s198, s2033}
+     - Moderate wasting with eodema, moderate wasting without oedema
+   * - SAM sequelae
+     - 
+     - {s199, s2036}
+     - Severe wasting with eodema, severe wasting without oedema
```
Comment (beatrixkh) [inline docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst:868]: It should be s199 - severe without oedema and s2036 = severe with oedema.

**Review by plinds — APPROVED** [review]
I'm approving because we need to get this merged today, but noting Abie and Nicole should read through once merged.

File: docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst [hunk 2]
```diff
@@ -0,0 +1,997 @@
...
+  :widths: 5 10
+  :header-rows: 1
+  
+  * - Condition
+    - Estimated by GBD sequelae
+  * - Kwashiorkor
+    - {Moderate wasting with oedema} + {Severe wasting with oedema}
+  * - Marasmus
+    - {Severe wasting without oedema} + {Severe wasting with oedema}
+
```
Comment (beatrixkh) [inline docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst:363]: add note -- other defns found in literature, this is GBD's defn.

**Pull request merged**